Tests/smokeTest.js

```diff
@@ -0,0 +1,41 @@
+var homepage = require('../Pages/homePage');
+var loginpage = require('../Pages/loginPage');
```

Comment:
use var a = ...,
             b = ..

instead of repeating var for every line here

Reply:
I changed this in code